server/static/css/main.css

```diff
@@ -58,4 +71,25 @@ body.transform {
   transform:rotate(180deg);
   -ms-transform:rotate(180deg);
   -webkit-transform:rotate(180deg);
-}
\ No newline at end of file
+}
+
+/* BACKGROUND */
```

Comment:
Bad idea, the main.css file is still small, we dont need one more http request just for the background. I think keeping background css code in main.css is great